doc/htmldoc/hpc/parallel_computing.rst

```diff
@@ -241,6 +241,31 @@ command should look like this
 Please refer to the documentation of your MPI implementation to learn
 more about the usage of ``mpirun``.
 
+
+.. important::
+
+    One should never call any ``nest.*`` function inside a block that will only be executed on a subset of MPI ranks.
+
+    Trying to access kernel information with a subset of MPI processes causes a deadlock.
+
+    For example:
+
+    **Don't do this**
+
+    .. code-block::
+
+      if nest.Rank() == 0:
+            print('RNG seed: {}'.format(
+                nest.rng_seed))
```

Comment:
```suggestion
      if nest.Rank() == 0:
            rng_seed = nest.rng_seed
            print(f"RNG seed: {rng_seed}")
```

I think for the purpose of illustration, the `nest.*` call should be on a a line of its own. This also makes the comparison with the next code block more clear. Moreover, we should use f-strings.